include/materials/bingham.tcc

```diff
@@ -15,6 +15,13 @@ mpm::Bingham<Tdim>::Bingham(unsigned id, const Json& material_properties)
     // Calculate bulk modulus
     bulk_modulus_ = youngs_modulus_ / (3.0 * (1. - 2. * poisson_ratio_));
 
+    // Special material properties
+    if (material_properties.contains("incompressible")) {
```

Comment:
I think we can make a `incompressible` boolean to be a compulsory input parameter just like the rest. What do you think?

Reply:
It's only needed for NS solver, so I recommend leaving it as is

Reply:
I am fine, but we have never done special input parameters for any `material` model. I just feel it's clearer if we have only compulsory input parameters. Less `if` statements too.

Reply:
It's only 1 if ezra